File: api/src/main/java/com/personalchef/api/model/Usuario.java
package com.personalchef.api.model;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Getter @Setter
public class Usuario {

  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  private Long id;

  @Lob
  private String foto;
  private String email;
  private String senha;
  private boolean ischef;
  private String descricao_chef;
  private String nome;
  private String sexo;
  private String endereco;
  private String complemento;
  private String telefone;
  private String nascimento;
  private Integer estrelas;
}

File: src/app/shared/model/usuario.ts
export class Usuario {
  id: number;
  email: string;
  senha: string;
  ischef: boolean;
  descricao_chef: string;
  foto: string;
  nome: string;
  sexo: string;
  endereco: string;
  complemento: string;
  telefone: string;
  nascimento: string;
  estrelas: number;
  estado_civil: string;
  conta_agencia: string;
  conta_numero: string;

  constructor() {
    this.email = '';
    this.senha = '';
    this.ischef = false;
    this.descricao_chef = '';
    this.foto = '';
    this.nome = '';
    this.sexo = 'M';
    this.endereco = '';
    this.complemento = '';
    this.telefone = '';
    this.nascimento = '';
    this.estrelas = null;
  }
}
